src/main/java/org/jenkinsci/plugins/workflow/cps/view/InterpolatedSecretsAction.java

```diff
@@ -0,0 +1,239 @@
+                }
+            } else {
+                failReason = "not an instance of FlowExecutionOwner.Executable";
+            }
+            throw new IllegalStateException(failReason);
+        }
+
+        @Exported
+        public List<String> getInterpolatedVariables() {
+            return interpolatedVariables;
+        }
+
+        @Override
+        public void onAttached(Run<?, ?> run) {
+            this.run = run;
+        }
+
+        @Override
+        public void onLoad(Run<?, ?> run) {
+            this.run = run;
+        }
+
+        @Override
+        public String getIconFileName() {
+            return null;
+        }
+
+        @Override
+        public String getDisplayName() {
+            return null;
+        }
+
+        @Override
+        public String getUrlName() {
+            return null;
+        }
+    }
+
+    private static String argumentToString(Object arg) {
+        String valueString;
+        if (arg instanceof Map.Entry) {
+            Map.Entry argEntry = (Map.Entry<String, Object>) arg;
+            Object value = argEntry.getValue();
+            if (value instanceof ArgumentsAction.NotStoredReason) {
+                switch ((ArgumentsAction.NotStoredReason) value) {
+                    case OVERSIZE_VALUE:
+                        valueString = "argument omitted due to length";
+                        break;
+                    case UNSERIALIZABLE:
+                        valueString = "unable to serialize argument";
+                        break;
+                    default:
+                        valueString = String.valueOf(value);
+                        break;
+                }
+            } else if (value instanceof Map || value  instanceof List || value instanceof UninstantiatedDescribable) {
+                valueString = argumentToString(value);
+            } else {
+                valueString = String.valueOf(value);
+            }
+            return argEntry.getKey() + ": " + valueString;
+        } else if (arg instanceof Map) {
+            valueString = ((Map<?, ?>) arg).entrySet().stream()
+                    .map(InterpolatedSecretsAction::argumentToString)
+                    .collect(Collectors.joining(", ", "[", "]"));
+        } else if (arg instanceof List) {
+            valueString = ((List<?>) arg).stream()
+                    .map(InterpolatedSecretsAction::argumentToString)
+                    .collect(Collectors.joining(", ", "[", "]"));
+        } else if (arg instanceof UninstantiatedDescribable) {
+            UninstantiatedDescribable ud = (UninstantiatedDescribable) arg;
+            Map<String, ?> udArgs = ud.getArguments();
+            if (ud.getSymbol() != null) {
+                valueString = udArgs.entrySet().stream()
+                        .map(InterpolatedSecretsAction::argumentToString)
+                        .collect(Collectors.joining(", ", "@" + ud.getSymbol() + "(", ")"));
```

Comment:
I don't think `@` should be here, for example in `DSLTest.complexSensitiveVariableInterpolationWithNestedDescribable` you didn't use it in the actual Pipeline, it only shows up in the output of this method.

```suggestion
                        .collect(Collectors.joining(", ", ud.getSymbol() + "(", ")"));
```

Reply:
yes, I was modeling after `UninstantiatedDescriable.toString()` which, as Jesse noted earlier, was not something I should be doing.  Will remove.

Reply:
removed in 49695e08dc0b1b386e1544805feba6b511120fc8